harmonic/ros2_spawn_model.md

```diff
@@ -43,9 +43,9 @@ within this tag. Here's an example:
   <gz_spawn_model 
     world="$(var world)"
     file="$(var file)"
-    xml_string="$(var xml_string)"
+    world_string="$(var world_string)"
```

Comment:
I just realized this change was merged recently, but I don't think it makes sense to call this `world_string` since this is being used to spawn a model. Mind changing this to `model_string` here and in ros_gz?

Reply:
Updated in [ab674e3](https://github.com/gazebosim/docs/pull/508/commits/ab674e3deacdbe85b112b1ae238697147f3f999d) and https://github.com/gazebosim/ros_gz/pull/605